src/Maestro/SubscriptionActorService/PullRequestActor.cs

```diff
@@ -872,8 +872,18 @@ await AddDependencyFlowEventsAsync(
                 MergePolicyCheckResult.PendingPolicies,
                 pr.Url);
 
+            var targetBranchDeps = (await darcRemote.GetDependenciesAsync(targetRepository, targetBranch)).ToList();
```

Comment:
can we be explicit with the type here instead of using `var`? It's not really obvious what `GetDependenciesAsync` returns